src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is started!");
```

Comment:
The message 'Truck is started!' is not very informative. It would be better to specify that it's the Truck's engine that has started, to provide more context.